src/instamatic/controller.py

```diff
@@ -13,13 +13,9 @@
 from instamatic.exceptions import TEMControllerError
 from instamatic.formats import write_tiff
 from instamatic.image_utils import rotate_image
-from instamatic.TEMController.microscope_base import MicroscopeBase
-
-from .deflectors import *
-from .lenses import *
-from .microscope import Microscope
-from .stage import *
-from .states import *
+from instamatic.microscope.base import MicroscopeBase
+from instamatic.microscope.components import *
```

Comment:
While you are at it, can you change this to more explicit imports to avoid the star import?

Reply:
Perhaps from `instamatic.microscopes import components` and then do `components.xxx` would be preferred over importing 20 classes into the global namespace.